compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/QueryExpressionNode.java

```diff
@@ -43,11 +43,14 @@ public QueryPipelineNode queryPipeline() {
 
     @Deprecated
```

Comment:
Not related to this PR. Small suggestion.
```suggestion
    /**
     * @deprecated This method is deprecated. Use {@link #resultClause()} instead.
     */
    @Deprecated
```

Reply:
Added